### ISSUE TITLE
Migrated the docker container to Balena Hub

### ISSUE THREAD

**Issue by maggie0002** [jaltekruse/Free-Math#213]
First of all, apologies for creating this as an issue, I couldn't find a better route. Feel free to close right away.

We worked together on that issue building Free-Math in a Docker container, and I published it on the Learner's Block. Wanted to let you know I have now published to the [Balena Hub](https://hub.balena.io/organizations/maggie0002/blocks/free-math) too. 

I won't be able to keep up with the updates for this though, would you mind if I submitted a pull request with a GitHub action to do the rebuilds on each release?

**Comment by jaltekruse** [comment]
Yeah I'd definitely take a pull request. Honestly, with the current structure of Free Math if seems a bit heavy to have a container for it, as there isn't any back-end I really should just publish the built files that anyone could throw onto a static server.

I know you had previously requested I start doing actual releases, that would help with some of this so you wouldn't have to tail master.

For the sake of simplifying this, would it be useful for you if I started cutting real releases and publishing a zip with the built site on github?

I don't know if anything else that is deployed on Learner's Block is just a static site, but this approach could make it more reasonable for those to all sit in one static web hosting container.
